exchanges/stream/websocket.go

```diff
@@ -85,12 +92,12 @@ func NewWebsocket() *Websocket {
 		// after subscriptions are made but before the connectionMonitor has
 		// started. This allows the error to be read and handled in the
 		// connectionMonitor and start a connection cycle again.
-		ReadMessageErrors: make(chan error, 1),
-		Match:             NewMatch(),
-		subscriptions:     subscription.NewStore(),
-		features:          &protocol.Features{},
-		Orderbook:         buffer.Orderbook{},
-		connections:       make(map[Connection]*ConnectionWrapper),
+		ReadMessageErrors:   make(chan error, 1),
+		Match:               NewMatch(),
+		subscriptions:       subscription.NewStore(),
+		features:            &protocol.Features{},
+		Orderbook:           buffer.Orderbook{},
+		connectionToWrapper: make(map[Connection]*ConnectionWrapper),
```

Comment:
I think this is regressing; Didn't we previously have it over-named and went just down to `connections` ?

Reply:
I don't remember to be honest, I think I had Connection Wrapper as Connection Candidate. 🤷 I will revert